cpp/simulations/abm_braunschweig.cpp

```diff
@@ -0,0 +1,267 @@
+#include <fstream>
+#include <vector>
+#include <iostream>
+#include "abm/abm.h"
+#include "memilio/io/result_io.h"
+#include "memilio/utils/uncertain_value.h"
+#include "boost/filesystem.hpp"
+#include "boost/algorithm/string/split.hpp"
+#include "boost/algorithm/string/replace.hpp"
```

Comment:
This looks like it isn't included in my boost.zip. Please confirm. Otherwise we can include it. I wonder why yours compiles.

Reply:
Same for me, it doesn't compile.

Reply:
Just to validate, do  you use `MEMILIO_USE_BUNDLED_BOOST=ON`?

Reply:
It is on for me.